integration_tests/client/metrics/test_detection.py

```diff
@@ -943,31 +955,31 @@ def test_evaluate_detection_with_label_maps(
             "type": "AR",
             "parameters": {"ious": [0.1, 0.6]},
             "value": 0.0,
-            "label": {"key": "class_name", "value": "maine coon cat"},
+            "label": {"key": "k2", "value": "v2"},
         },
         {
             "type": "AR",
             "parameters": {"ious": [0.1, 0.6]},
             "value": 0.0,
-            "label": {"key": "k2", "value": "v2"},
+            "label": {"key": "class_name", "value": "maine coon cat"},
         },
         {
             "type": "AR",
             "parameters": {"ious": [0.1, 0.6]},
             "value": 0.0,
-            "label": {"key": "class", "value": "siamese cat"},
+            "label": {"key": "class", "value": "british shorthair"},
         },
         {
             "type": "AR",
             "parameters": {"ious": [0.1, 0.6]},
-            "value": 0.5,
-            "label": {"key": "k1", "value": "v1"},
+            "value": -1.0,
+            "label": {"key": "class", "value": "cat"},
         },
         {
             "type": "AR",
             "parameters": {"ious": [0.1, 0.6]},
-            "value": -1.0,
-            "label": {"key": "class", "value": "cat"},
+            "value": 0.0,
+            "label": {"key": "class", "value": "siamese cat"},
```

Comment:
note that the order of some of these expected_metrics has changed, but the values themselves are the same as what they were before.